source/index.js

```diff
@@ -13,12 +14,43 @@ renderer.roundPixels = true
 document.getElementById("mount").appendChild(renderer.view)
 window.game = new Pixi.Container()
 game.addChild(new Junkership())
+game.score=0
 
 game.addChild(new Trashbot(Reference.GAME_WIDTH, Reference.GAME_HEIGHT/2))
 
 var loop = new Afloop(function(delta) {
     game.children.forEach((child) => {
         child.update(delta)
+
+        if (child instanceof Trashbot ||
+            child instanceof Junkership){
+            collisionCheck(child)
+        }
```

Comment:
Hmm.. I still feel that we should perform the individual collisions within the classes, not in the main loop. Thoughts from the rest of the team? @dhowerton, @HolySchmitt, @bonfortune.